kajiki/ir.py

```diff
@@ -267,6 +272,42 @@ def py(self):
         yield self.line("elif local.__kj__.case(%s):" % self.decl)
 
 
+class SPMNode(HierNode):
+    """Structural Pattern Matching Node"""
+
+    def __init__(self, decl, *body):
+        super().__init__(body)
+        self.decl = decl
+
+    def py(self):
+        yield self.line("match %s:" % self.decl)
```

Comment:
Can we wrap the value in parenthesis to rule out the possibility of any syntax foolery (e.g., a newline in `self.decl`)?

Reply:
Thanks!